src/test/parser.test.ts

```diff
@@ -609,6 +609,18 @@ suite('JSON Parser', () => {
 
 		assert.strictEqual(semanticErrors!.length, 0);
 
+		semanticErrors = validate2(jsonDoc, textDoc, {
+			type: 'object',
+			properties: {
+				"one": {
+					type: 'string',
+					pattern: 'invalid pattern',
+				}
+			}
+		});
+
+		assert.strictEqual(semanticErrors!.length, 1);
```

Comment:
I'm not entirely sold on the need for this test (or at least, how it functions).
Let me know what you think.